src/main/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCaller.scala

```diff
@@ -117,6 +151,11 @@ class VanillaUmiConsensusCaller
   private val NotEnoughReadsQual: PhredScore = 0.toByte // Score output when masking to N due to insufficient input reads
   private val TooLowQualityQual: PhredScore = 2.toByte  // Score output when masking to N due to too low consensus quality
 
+  // var to track how many reads meet various fates
+  private var _totalReads: Long = 0
+  private var _filteredForMinReads: Long = 0
+  private var _filteredForMinorityAlignment: Long = 0
```

Comment:
Agreed, but a class seems overkill.  I think this is more or less convention when defining read-only vars since the var and the method name cannot match.